model/src/main/java/com/iabtcf/v2/OutOfBandConsent.java

```diff
@@ -30,4 +32,106 @@
 	 * @return all vendors that the publisher permits to use OOB legal bases
 	 */
 	IntStream getAllAllowedVendors();
+
+	static Builder newBuilder() {
+		return new Builder();
+	}
+
+	static Builder newBuilder(OutOfBandConsent oob) {
+		return new Builder(oob);
+	}
+
+	class Builder {
+
+		private BitSet disclosedVendors = new BitSet();
```

Comment:
can probably save some space here and in other builders by using a static empty bit set until something is added to these fields.

Reply:
will do later